R/write.R

```diff
@@ -58,3 +84,36 @@ output_column.POSIXt <- function(x) {
   format(x, "%Y-%m-%dT%H:%M:%OSZ", tz = "UTC")
 }
 
+#' Write a single R object to file
+#'
+#' Consistent wrapper around \code{\link{saveRDS}}. \code{write_rds} never
+#' compresses by default as space is generally cheaper than time.
+#'
+#' @section guess compression:
+#' N.B. this is not yet implemented but some empirical testing may help come up
+#' with something useful.
+#' @param x R object to write to serialise.
+#' @param path Path to write to.
+#' @param compress Compression method to use one of \code{c("none", "gz" ,"bz",
+#' "xz")}.
+#' @param ... Additional arguments to connection function.
```

Comment:
"For example, `compression_level`, allows you to control the space-time tradeoff for the compression methods)